### DISCUSSION TITLE
pip_add + trusted host

### DISCUSSION THREAD

**Discussion by gyansinha** [JuliaPy/CondaPkg.jl#94] [Q&A] [answered]
what's the equivalent of this command for conda pip_add?

pip install --trusted-host pypi.org --trusted-host pypi.python.org --trusted-host files.pythonhosted.org <package>

**Comment by cjdoris** [comment, marked as answer]
This can't currently be done at the command line. I think setting the env var `PIP_TRUSTED_HOST="pypi.org pypi.python.org files.pythonhosted.org"` will do it, or you can edit your pip.conf file.

**Reply by gyansinha** [reply]
thanks - can you tell me where the pip.conf file is located within the CondaPkg directory layout?

**Reply by cjdoris** [reply]
pip.conf is in a global location (https://pip.pypa.io/en/stable/topics/configuration/) so only use it if you want to configure pip across your system.

**Reply by gyansinha** [reply]
Thanks - the environment variable worked. Much appreciated.
